tests/e2e/common.go

```diff
@@ -820,7 +820,26 @@ func (td *OsmTestData) Cleanup(ct CleanupType) {
 		}
 
 		for _, ns := range testNs.Items {
-			err := td.DeleteNs(ns.Name)
+			// Delete Helm releases created in the namespace
```

Comment:
Question. Would it be help if this was run defacto on `DeleteNs` code? Just in case someone decided to manually run a `DeleteNs` individually (as part of the test) on an NS that had helm resources.

Ok eitherway for the time being, ofc

Reply:
Good call. I'll make that change.